pkg/kclient/kclient.go

```diff
@@ -55,6 +55,7 @@ type Client struct {
 	// Is server side apply supported by cluster
 	// Use IsSSASupported()
 	isSSASupported                     *bool
+	checkIngressSupports               bool
```

Comment:
Please add a comment describing the variable.

Reply:
Adding